verification/testNF90io/input/gendata.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+import numpy.matlib as matlib
+import os
+import shutil
+
+H = 2000.
+h0=500.
+N0 = 5.2e-3
+
+outdir='../run/'
+try:
+  print('Making %s'%outdir)
+  os.mkdir(outdir)
+except:
+  print (outdir+' Exists')
+
+shutil.copy('./gendata.py',outdir)
+
+# These must match ../code/SIZE.h
+ny = 2*6
+nx = 4*4
+nz = 6
+
+
+dy = 1000.
+dx = 1000.
+
+dx = np.zeros(nx)+dx
+x=np.cumsum(dx)
+x = x-x[int(nx/2)]
+
+with open(outdir+"/delXvar.bin", "wb") as f:
+  dx.tofile(f)
```

Comment:
This writes native-endian.  Can you change to big-endian here and below, e.g.,

```python
    dx.astype('>f8').tofile(f)
```

Reply:
Yeah, I'll do it, but reluctantly... At some point it'd be nice to phase out big-endian, unless VAX makes a comeback someday  :wink: